src/components/BooksForm.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const categories = ['Action', 'Biography', 'History', 'Horror',
+    'Kids', 'Learning', 'Sci-Fi'];
+
+  const renderOption = category => (
+    <option key={category} value={category}>
+      {category}
+    </option>
+  );
+
+  return (
+    <form>
+      <input placeholder="Book title" />
+      <select>{categories.map(category => renderOption(category))}</select>
+      <button type="submit">Submit</button>
+    </form>
+  );
+};
+
+export default BooksForm;
```

Comment:
this is a "container" component, pleace is inside `src/containers/` folder 😊
(meaning you will have to change couple IMPORTS where this component is used)

Reply:
Also initialise state here with `title` and `category`